src/lib/sap_ha/wizard/scenario_selection_page.rb

```diff
@@ -42,8 +42,12 @@ def set_contents
       end
 
       def refresh_view
-        previous_configs = SapHA::Helpers.get_configuration_files(@model.product_id)
-        previous_configs_popup(previous_configs) if !previous_configs.empty?
+        begin
+          previous_configs = SapHA::Helpers.get_configuration_files(@model.product_id)
+          previous_configs_popup(previous_configs) if !previous_configs.empty?
+	rescue StandardError => e
+          log.info "Could not parse previous config files: #{e.message}"
```

Comment:
NP: I would probably use more specific exception and warning level. But nothing critical